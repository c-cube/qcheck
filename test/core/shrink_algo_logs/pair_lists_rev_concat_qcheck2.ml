open QCheck2

let pair_lists_rev_concat =
  Test.make ~name:"pairs lists rev concat"
    Gen.(pair (list (pint ~origin:0)) (list (pint ~origin:0)))
    (Log.shrinks Print.(pair (list int) (list int)) @@
       fun (xs,ys) -> List.rev (xs@ys) = (List.rev xs)@(List.rev ys))
;;
try Test.check_exn pair_lists_rev_concat with Test.Test_fail _ -> ()
