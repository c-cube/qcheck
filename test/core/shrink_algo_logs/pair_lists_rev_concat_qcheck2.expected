fails ([746128036134350390; 3320138915538235609; 762885526058710783; 2576722397014108197; 2212906866981369341; 1640905365439660237; 1332813644634888458; 4200732135473036391; 4019205362601190422; 3604960216201531334; 1030920758746055813; 42698023673707078; 461254803691789422; 3907752289102763354; 2376625294767365140; 233912705210533500; 2885984270693699778; 206688355204064373; 3125861600804315502; 995060844299483603; 3304051089873692429; 2844386052677950810; 783948934796125836; 2811664358749514936; 577396296345547726; 4034593187793338681; 2876338321387969175; 87742541397720461; 3913144465737681491; 2157897674888391252; 1900045916102699699; 470374935996346701; 1058614341825424502; 1559319129724948356; 3213658892819147890; 3499789205307451184; 2949893178354239875; 4081767524822084600; 3498215066845906710; 3888114807866831417; 1617765899064123599; 222148284913888236; 1727736514015983487; 3627298274602852316; 3898699378202599689; 2325974463411213714; 736613994040121347; 3414272520188964400; 2385629250013173880; 1422734660007259092; 4444687725003531440; 457048644972362124; 4167435680361369008; 3782659039753844321; 3535545447366295718; 1427021198443879304; 1780554213351577732; 2339368131399213148; 2751299455197144887; 2600977953485488800; 2162723098335521495; 1983575711044503816; 2874018304296889121; 1748110412865819786; 3323702044602342403; 1051679191852119048; 2091273115503484635; 1494934802956191728; 2924085192404355751; 3504533652220676065; 1021194546620295917; 1475531882113471101; 2107955955816427122; 4212121560373082225; 469399435806458576; 2088479218423533002; 1863269600650047234; 2442470379891932470; 2578459975428991434; 3314506194157773646; 2099966072041048397; 4345233542768063678; 2336561938922647935; 3244060950469953135; 4079854335523721792; 2828045190569874047; 1785372679127697414; 4540055182899246774; 1752752047792425988; 701266008688472284; 3366925537143040935; 1598238962204476610; 2927976139793784142; 323035789210505289; 151715852996735469], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([884212182938570936; 613425671695126647; 4512637923211922161; 486685375135801800; 641056412095441698; 3750490204890178363; 3145756734976380312; 1961342163929721306; 4201588351799856356; 1998192899672709722; 862266431350240130; 3328418251958078567; 4429552149330209394; 4305699655940863834; 182560322506877800; 3126796769256439285; 2439427584254951601; 342933758871195373; 3427434385086153570; 13134053656904240; 3586910973165718204; 4455552537819661014; 32445640770685866; 2278623161446483710; 1632903327713615466; 2900144449389416574; 345670844570608874; 2179211411668797418; 851383051062622718; 2243784010337392710; 2397609872536246025; 1286008752150352513; 2494253037302305383; 1069672522808247007; 1218081170985299468; 1549896317694330907; 2427590248535557030; 1983847614663587292; 421523309653103579; 2007583531761632547; 1768794228951364600; 2731279530478149511; 3698468194261084927; 2326388082144332864; 2109163211959572459; 952431139506660731; 1999177455861418171], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([466090079215598704; 114963491218766700; 1870448274228386459; 1079057410209247991; 2878201886415698988; 3329126812201254532; 2129642638309123927; 3146139493280747422; 448789927510083313; 2301527649898309728; 3746708087324583722; 1997630303591956075; 545014510072175503; 1949977473874760355; 2418572032569070867; 2529234444504398903; 3907126064040108583; 3536671064542785126; 2684061773139892207; 2185913413220317925; 3420412244091129264; 856218886446940986; 3484150691150522657], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2068575257510797379; 2803165160977021446; 274603623993075597; 3993419239738349511; 160656567065795743; 2739431745249531050; 2314260104205866360; 177555709967860887; 4607914717083331092; 2729619983524776475; 3322709682347690803], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2932241091789215198; 2861543521634873622; 2336231494816689504; 1460370162771829994; 232062967338649209], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([3762171117042495591; 4588024816217148396], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([2152069955941623198], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
fails ([0], [2696939011544317271; 1975781842156211841; 1035416029544138122; 1118378519987614091])
holds ([0], [])
fails ([0], [1599388225294475516; 3378876932193098527])
holds ([0], [])
fails ([0], [794526490094798868])
holds ([0], [])
holds ([0], [0])
fails ([0], [397263245047399434])
holds ([0], [0])
fails ([0], [198631622523699717])
holds ([0], [0])
fails ([0], [99315811261849858])
holds ([0], [0])
fails ([0], [49657905630924929])
holds ([0], [0])
fails ([0], [24828952815462464])
holds ([0], [0])
fails ([0], [12414476407731232])
holds ([0], [0])
fails ([0], [6207238203865616])
holds ([0], [0])
fails ([0], [3103619101932808])
holds ([0], [0])
fails ([0], [1551809550966404])
holds ([0], [0])
fails ([0], [775904775483202])
holds ([0], [0])
fails ([0], [387952387741601])
holds ([0], [0])
fails ([0], [193976193870800])
holds ([0], [0])
fails ([0], [96988096935400])
holds ([0], [0])
fails ([0], [48494048467700])
holds ([0], [0])
fails ([0], [24247024233850])
holds ([0], [0])
fails ([0], [12123512116925])
holds ([0], [0])
fails ([0], [6061756058462])
holds ([0], [0])
fails ([0], [3030878029231])
holds ([0], [0])
fails ([0], [1515439014615])
holds ([0], [0])
fails ([0], [757719507307])
holds ([0], [0])
fails ([0], [378859753653])
holds ([0], [0])
fails ([0], [189429876826])
holds ([0], [0])
fails ([0], [94714938413])
holds ([0], [0])
fails ([0], [47357469206])
holds ([0], [0])
fails ([0], [23678734603])
holds ([0], [0])
fails ([0], [11839367301])
holds ([0], [0])
fails ([0], [5919683650])
holds ([0], [0])
fails ([0], [2959841825])
holds ([0], [0])
fails ([0], [1479920912])
holds ([0], [0])
fails ([0], [739960456])
holds ([0], [0])
fails ([0], [369980228])
holds ([0], [0])
fails ([0], [184990114])
holds ([0], [0])
fails ([0], [92495057])
holds ([0], [0])
fails ([0], [46247528])
holds ([0], [0])
fails ([0], [23123764])
holds ([0], [0])
fails ([0], [11561882])
holds ([0], [0])
fails ([0], [5780941])
holds ([0], [0])
fails ([0], [2890470])
holds ([0], [0])
fails ([0], [1445235])
holds ([0], [0])
fails ([0], [722617])
holds ([0], [0])
fails ([0], [361308])
holds ([0], [0])
fails ([0], [180654])
holds ([0], [0])
fails ([0], [90327])
holds ([0], [0])
fails ([0], [45163])
holds ([0], [0])
fails ([0], [22581])
holds ([0], [0])
fails ([0], [11290])
holds ([0], [0])
fails ([0], [5645])
holds ([0], [0])
fails ([0], [2822])
holds ([0], [0])
fails ([0], [1411])
holds ([0], [0])
fails ([0], [705])
holds ([0], [0])
fails ([0], [352])
holds ([0], [0])
fails ([0], [176])
holds ([0], [0])
fails ([0], [88])
holds ([0], [0])
fails ([0], [44])
holds ([0], [0])
fails ([0], [22])
holds ([0], [0])
fails ([0], [11])
holds ([0], [0])
fails ([0], [5])
holds ([0], [0])
fails ([0], [2])
holds ([0], [0])
fails ([0], [1])
holds ([0], [0])
